prymer/primer3/primer3.py

```diff
@@ -228,13 +227,23 @@ def __init__(
         self,
         genome_fasta: Path,
         executable: Optional[str] = None,
-        variant_lookup: Optional[VariantLookup] = None,
+        vcfs_or_lookup: Optional[list[Path] | VariantLookup] = None,
+        use_cache: bool = False,
+        min_maf: float = 0.0,
+        include_missing_mafs: bool = False,
     ) -> None:
         """
         Args:
             genome_fasta: Path to reference genome .fasta file
             executable: string representation of the path to primer3_core
-            variant_lookup: VariantLookup object to facilitate hard-masking variants
+            vcfs_or_lookup: an optional list of VCF files or `VariantLookup` object
+                with which to hard-mask variants
+            use_cache: whether to use an in-memory `VariantLookup` for fast querying
+            min_maf: an optional minimum Minor Allele Frequency with which to filter a list of
+                VCF files (return only variants with at least this minor allele frequency)
+            include_missing_mafs: when filtering variants with a minor allele frequency,
+                `True` to include variants with no annotated minor allele frequency, otherwise
+                `False`. If no minor allele frequency is given, then this parameter does nothing.
```

Comment:
```suggestion
```

I agree with @msto that we were going to leave these out, and users can use `VariantLookup.build()` if they want more control.